backend/src/model/entity.rs

```diff
@@ -11,70 +11,224 @@ use diesel::{Identifiable, Insertable, Queryable};
+    pub habitat: Option<String>,
+    /// Fetched from Permapeople as \`spacing\` and from Reinsaat as Distances and renamed. (fetched from Permapeople and Reinsaat)
+    pub spacing_en: Option<String>,
+    /// Fetched from Permapeople as \`wikipedia\` and renamed. (fetched from Permapeople)
+    pub wikipedia_url: Option<String>,
+    /// (fetched from Permapeople)
+    pub days_to_maturity: Option<String>,
+    /// (fetched from Permapeople)
+    pub pests: Option<String>,
+    /// The version of the entry. (fetched from Permapeople)
+    pub version: Option<String>,
+    /// (fetched from Permapeople)
+    pub germination_time: Option<String>,
+    /// The description of the entry. (fetched from Permapeople)
+    pub description: Option<String>,
+    /// The permapeople id of the parent entry pointing to the `external_id` column. (fetched from Permapeople)
+    pub parent_id: Option<String>,
+    /// Enum value indicating the source of the entry.
+    pub external_source: Option<ExternalSource>,
+    /// The external id of the entry used in combination with the `external_source` column.
+    pub external_id: Option<String>,
+    /// The external URL provided by the origin source.
+    pub external_url: Option<String>,
+    /// Fetched from PracticalPlants as \`root_zone_tendency\` and merged with root_depth of Permapeople. (fetched from PracticalPlants and Permapeople)
+    pub root_depth: Option<String>,
+    /// The article number of the plant in the Reinsaat database. (fetched from Reinsaat)
+    pub external_article_number: Option<String>,
+    /// (fetched from Reinsaat)
+    pub external_portion_content: Option<String>,
+    /// Fetched from Reinsaat as \`Direktsaat\` and renamed. (fetched from Reinsaat)
+    pub sowing_outdoors_de: Option<String>,
+    /// String array of numbers representing a time period. The year is divided into 24 periods of half a month each. For example "\[8,9,10\]" means from the 2nd half of April to the 2nd half of May incl. (fetched from Reinsaat)
+    pub sowing_outdoors: Option<Vec<Option<i16>>>,
+    /// String array of numbers representing a time period. The year is divided into 24 periods of half a month each. For example "\[8,9,10\]" means from the 2nd half of April to the 2nd half of May incl. (fetched from Reinsaat)
+    pub harvest_time: Option<Vec<Option<i16>>>,
+    /// (fetched from Reinsaat)
+    pub spacing_de: Option<String>,
+    /// (fetched from Reinsaat)
+    pub required_quantity_of_seeds_de: Option<String>,
+    /// (fetched from Reinsaat)
+    pub required_quantity_of_seeds_en: Option<String>,
+    /// (fetched from Reinsaat)
+    pub seed_planting_depth_de: Option<String>,
+    /// (fetched from Reinsaat)
+    pub seed_weight_1000_de: Option<String>,
+    /// (fetched from Reinsaat)
+    pub seed_weight_1000_en: Option<String>,
+    /// Fetched from Permapeople as \`1000_seed_weight_g\` and renamed. (fetched from Permapeople)
+    pub seed_weight_1000: Option<f64>,
```

Comment:
For some plants some or all of them are missing, e.g. https://www.reinsaat.at/shop/DE/paprika/spitzpaprika_1/kurtovska_kapiya_1/
Even though they are present (Tausendkorngewicht (TKG): 7.15 g)

Other example: https://www.reinsaat.at/shop/DE/paprika/spitzpaprika_1/shintokiwa/ does not have `seed_weight_100` even though present Tausendkorngewicht (TKG): 24.54 g (but has _de and _en)